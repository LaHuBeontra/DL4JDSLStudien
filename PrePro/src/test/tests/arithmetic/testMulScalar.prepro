function main() {
	import scal numbers;

	print numbers;

	scal result = numbers * numbers * numbers;

	export result;
}
