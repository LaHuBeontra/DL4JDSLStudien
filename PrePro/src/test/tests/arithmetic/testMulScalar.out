DefaultPreProDataSet{
dataSetEntries=
[DataSetEntry{variableName='result', variableShape=[4, 1], startColumn=0, length=1}],
dataSet=
[1.00,  8.00,  27.00,  64.00]

constants={}

calculated_variables=
result=[1.00,  8.00,  27.00,  64.00]
}